paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -212,11 +212,10 @@ internal class CustomerSheetViewModel @Inject constructor(
     private fun onItemRemoved(paymentMethod: PaymentMethod) {
         viewModelScope.launch {
             customerAdapter.detachPaymentMethod(paymentMethodId = paymentMethod.id!!)
-                .onFailure {
+                .onFailure { throwable ->
                     updateViewState<CustomerSheetViewState.SelectPaymentMethod> {
-                        // TODO (jameswoo) translate error message
                         it.copy(
-                            errorMessage = "Unable to remove payment method",
+                            errorMessage = throwable.message,
```

Comment:
I'm still not convinced we should do this.

Reply:
Exception messages aren't localized, and typically not intended for end users. Some might be, but it's easy to mess this up, and I think it's full of foot guns for long term maintenance.

Reply:
The errors are coming from these sources:
- Stripe API, which has localized error messages
- The merchants server, which they should localize themselves
- Or some localized errors from the SDK itself (for example, networking errors)

This is what iOS is doing, but if you feel strongly about this, we should align with iOS and have the same error messages. Thoughts @wooj-stripe?